cmd/juju/commands/upgradecontroller.go

```diff
@@ -121,7 +116,21 @@ func (c *upgradeControllerCommand) getControllerAPI() (ControllerAPI, error) {
 	return c.NewControllerAPIClient()
 }
 
+func (c *upgradeControllerCommand) getModelManagerAPI() (ModelManagerAPI, error) {
```

Comment:
This is already on `upgradeJujuCommand` - that just needs to be moved to baseUpgradeCommand instead of introducing a new implementation here

Reply:
I don't think we can move that from upgradeJujuCommand to baseUpgradeCommand unless we want to move the modelcmd.ModelCommandBase from upgradeJujuCommand to the upgradeJujuCommand as well.